.changeset/dull-sloths-clap.md

```diff
@@ -0,0 +1,23 @@
+---
+'@backstage/plugin-catalog-backend-module-gitlab-org': minor
```

Comment:
both look like a `patch` as there will be no breaking change for the adopters, right?

I would suggest splitting this changeset into two separate changesets for each of the packages so that the CHANGELOG for each package will contain only the appropriate changes.

Reply:
Done! :)